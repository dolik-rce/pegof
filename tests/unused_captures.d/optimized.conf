optimize all
input unused_captures.d/captures.peg
