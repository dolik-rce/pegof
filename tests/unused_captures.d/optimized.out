%value "char*"

X <- (A) <B> <C> $2 { printf("B = %s\n", $1) }

A <- "A" { $$ = "a"; }

B <- "B" { $$ = "b"; }

C <- "C" { $$ = "c"; }
